db/migrate/20210219043102_add_check_constraint_for_username.rb

```diff
@@ -0,0 +1,12 @@
+class AddCheckConstraintForUsername < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured do
+      execute(<<~SQL.squish)
+        ALTER TABLE "users"
+        ADD CONSTRAINT "users_username_null"
```

Comment:
should the name. be `users_username_not_null`? It feels a bit clearer by expressing what the check does

Reply:
I followed the example from `strong_migrations ` here, but I agree, the other name is better. Will update.

Reply:
Done ✅